src/api/app/views/webui/request/_superseded_by_message.html.haml

```diff
@@ -3,28 +3,26 @@
     .ui-state-info.ui-corner-all.ui-widget-shadow.padding-10pt#request-supersedes-hint
       %span.ui-icon.ui-icon-info
         This request supersedes:
-        - superseding.each do |bs_request|
+        - superseding.each_with_index do |bs_request, index|
           = link_to "request #{bs_request.number}", request_show_path(number: bs_request.number)
-          (
-          = link_to 'Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number)
-          )
+          = surround '(', ')' do
+            = link_to 'Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number)
+          = ', ' if index < superseding.length - 1
```

Comment:
@eduardoj didn't you want to add an `and`? :rofl:

Reply:
I would have used `superseding.last.equal? bs_request` instead of having the index :thinking: